command/pr_test.go

```diff
@@ -304,18 +305,48 @@ func TestPRList(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	eq(t, output.Stderr(), `
+	assert.Equal(t, `
 Showing 3 of 3 pull requests in OWNER/REPO
 
-`)
-	eq(t, output.String(), `32	New feature	feature
-29	Fixed bad bug	hubot:bug-fix
-28	Improve documentation	docs
-`)
+`, output.Stderr())
+
+	lines := strings.Split(output.String(), "\n")
+	res := []*regexp.Regexp{
+		regexp.MustCompile(`#32.*New feature.*feature`),
+		regexp.MustCompile(`#29.*Fixed bad bug.*hubot:bug-fix`),
+		regexp.MustCompile(`#28.*Improve documentation.*docs`),
```

Comment:
Seeing these regexps makes me think that we could benefit in the future from an assertion matcher that first strips ANSI escape sequences. Then we can assert expected output in an exact manner (i.e. without `.*`)

Reply:
yeah, I think that would be wise.